doc/Makefile

```diff
@@ -51,7 +51,7 @@ pdf: deps
 	$(JULIA_EXECUTABLE) --color=yes $(call cygpath_w,$(SRCDIR)/make.jl) -- pdf $(DOCUMENTER_OPTIONS)
 	@echo "Build finished."
 
-# The deploy target should only be called in Travis builds
+# The deploy target should only be called in CI builds
```

Comment:
Is this even correct? Is this target still used? I have no idea how the Julia CI works in general, so I can't judge this. If it is not used, then perhaps it'd be better to remove the whole target.